tests/pw/.env

```
@@ -0,0 +1,50 @@
+DB_USER_PASSWORD=password
+DATABASE=tests-wordpress
+DB_PORT=9998
+DB_PREFIX=wp
+
+# REST API - CRITICAL: Query string format for Docker!
+SERVER_URL=http://localhost:9999/?rest_route=
+
+# Test Data (auto-populated by setup scripts)
+CUSTOMER_ID=3
+VENDOR_ID=4
+CUSTOMER2_ID=5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CUSTOMER2_ID key should go before the CUSTOMER_ID key

(UnorderedKey)